api/serverV2.go

```diff
@@ -119,3 +120,9 @@ func (svr *ServerV2) Stop() error {
 	}
 	return nil
 }
+
+// ReceiveBlock receives the new block
+func (svr *ServerV2) ReceiveBlock(blk *block.Block) error {
+	svr.core.readCache.Clear()
+	return svr.core.chainListener.ReceiveBlock(blk)
```

Comment:
if both `readCache` & `chainListener` are variables of `core`, we may implement this function in core?